WalletWasabi.Gui/Program.cs

```diff
@@ -58,10 +68,17 @@ private static void Main(string[] args)
 					BuildAvaloniaApp().StartShellApp("Wasabi Wallet", AppMainAsync, args);
 				}
 			}
+			catch (OperationCanceledException ex)
```

Comment:
What can throw this exception?

Reply:
I have no specific answer to this but in many cases in the underlying objects like Pwfinder and Mix handled this exception separately - so I do.

Reply:
The thing is that once you add this, nobody will be confident to remove it even if it is not needed. So there should probably be a reason to add in the first place.

Reply:
You are right about this - that it is hard to prove that this is required. But do we want to risk if there is an accidental leak of OperationCanceledException when Disposing objects, Wasabi will show a crash report to the user? Maybe it is not there now but later it can get into the code accidentally. This would guarantee that in that case, the exception won't scare the user.

Reply:
From this point of view, probably yes.

Reply:
I got one. Global.InitializeNoWalletAsync() can throw.